parent/pom.xml

```diff
@@ -426,18 +441,25 @@
         </plugin>
       </plugins>
     </pluginManagement>
-    <!-- Execute Bonita artifacts validation -->
     <plugins>
       <plugin>
         <groupId>org.bonitasoft.maven</groupId>
         <artifactId>bonita-project-maven-plugin</artifactId>
         <executions>
+          <!-- Execute Bonita artifacts validation -->
           <execution>
             <id>validate-artifacts</id>
             <goals>
               <goal>validate</goal>
             </goals>
           </execution>
+          <!-- Copy Bonita provided pages if needed -->
+          <execution>
+            <id>copy-provided-pages</id>
```

Comment:
:warning: The execution goal should only be defined in the `app` module of a bonita-project (done at project creation/migration in the Studio), otherwise, each submodule will execute it.